template/vars/taskcluster_version_fxci.yaml

```diff
@@ -0,0 +1,3 @@
+# This defines the current Taskcluster version, the default version for worker-runner and workers.
+env_vars:
+  TASKCLUSTER_VERSION: 38.0.5
```

Comment:
I think this config is for gcp only, is that right? The name suggests it isn't specific to a cloud - so maybe needs to be `template/vars/default_gcp_firefoxci.yaml` ?

Although if this is related to the other issue where v4l2loopback needs to be built from source, i.e. there is a certain condition when you always need to do _both_ of the following:
*) build v4l2loopback from source
*) deploy version 38.0.5 of taskcluster workers
then maybe it would be better to create a single config for that condition, with both options specified, and name it _after the condition_ that requires it, e.g. `bug_1234567_overlays.yaml` rather than name it after the deployment etc, and then list the new config in `builders/docker_firefoxci_gcp_l1.yaml` and `builders/docker_firefoxci_gcp_l3.yaml`?

I _believe_ that configs are listed in precedence order, and therefore by adding that config file at the start (or at the end, not sure which) of the list, its values can take precedence. When that condition goes away, it is then simple to remove the patch, and at a glance it is easy to see that these two conditions are directly related.

Anyway, if these two changes aren't related to each other (holding back the taskcluster version number, and building vl412loopback from source) then I think just changing the name of the yaml file is sufficient to include gcp in the name.